TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Induction.cshtml.cs

```diff
@@ -0,0 +1,93 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using TeachingRecordSystem.Core.DataStore.Postgres;
+using TeachingRecordSystem.Core.Dqt.Models;
+using TeachingRecordSystem.Core.Dqt.Queries;
+
+namespace TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail;
+
+public class InductionModel(TrsDbContext dbContext, ICrmQueryDispatcher crmQueryDispatcher, IClock clock) : PageModel
+{
+    private static readonly string NoQualifiedTeacherStatusWarning = "This teacher has not been awarded QTS and is therefore ineligible for induction.";
+    private static readonly string InductionIsManagedByCpdWarning = "To change a teacher\u2019s induction status to passed, failed, or in progress, use the Record inductions as an appropriate body service.";
+    private bool StatusIsManagedByCPD;
+    private bool TeacherHoldsQualifiedTeacherStatus;
+
+    [FromRoute]
+    public Guid PersonId { get; set; }
+
+    public InductionStatus Status { get; set; }
+
+    public DateOnly? StartDate { get; set; }
+
+    public DateOnly? CompletionDate { get; set; }
+
+    public InductionExemptionReasons ExemptionReasons { get; set; }
+
+    public bool ShowStartDate =>
+        Status is InductionStatus.Failed
+        or InductionStatus.FailedInWales
+        or InductionStatus.InProgress
+        or InductionStatus.Passed;
```

Comment:
Is this the same as:
```suggestion
    public bool ShowStartDate => Status.RequiresStartDate();
```

Reply:
oh man, I didn't see those!